src/test/java/net/snowflake/client/core/SFArrowResultSetIT.java

```diff
@@ -41,16 +43,24 @@
 
 @Category(TestCategoryArrow.class)
 public class SFArrowResultSetIT {
+
+  /** Necessary to conditional ignore tests */
+  @Rule public ConditionalIgnoreRule rule = new ConditionalIgnoreRule();
+
   private Random random = new Random();
 
-  /** allocator for arrow */
+  /**
+   * allocator for arrow RootAllocator is shaded so it cannot be overridden when testing thin or fat
```

Comment:
Does it mean that the customers can't use it too?

Reply:
those classes are shaded so if we are testing thin jar (or fat jar) then those classes have different package and customers should use this class but with different package